scripts/run_docker_build.sh

```diff
@@ -18,14 +18,19 @@ CONDARC
 )
 
 cat << EOF | docker run -i \
-                        -v ${REPO_ROOT}/recipes:/conda-recipes \
+	                -v ${REPO_ROOT}:/staged-recipes \
```

Comment:
Had to have access to the repo for the `git` trick to work. So mounted the whole repo instead of just the recipes.